src/applications/mhv-medical-records/containers/ccdContent/OHOnlyIntroText.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const OHOnlyIntroText = () => (
+  <>
+    <h1>Download your medical records report</h1>
+    <p className="vads-u-margin--0">
```

Comment:
Remove the className so it’s just a <p> -this ensures proper spacing before the yellow alert.